class/defaults.yml

```diff
@@ -1,4 +1,32 @@
 parameters:
   alerts_exporter:
     =_metadata: {}
-    namespace: syn-alerts-exporter
+    namespace: appuio-alerts-exporter
+
+    namespaceMetadata:
+      annotations:
+        openshift.io/node-selector: "node-role.kubernetes.io/infra="
+      labels:
+        openshift.io/cluster-monitoring: "true"
+
+    images:
+      alerts_exporter:
+        registry: ghcr.io
+        image: appuio/alerts-exporter
+        tag: main
```

Comment:
I guess we'll update this to a real tag afterwards?

Reply:
Done in https://github.com/appuio/component-alerts-exporter/pull/1/commits/5c2996f22df628228cec4943ad5961a206c9c67f